ast/ast.go

```diff
@@ -1264,7 +1265,7 @@ type TableSample struct {
 
 // TableSampleSize is size part of TABLESAMPLE clause.
 //
-//	({{.Value | sql}} {{.Unit}})
+//	({{.Value | sql}} {{.DatePartName}})
```

Comment:
What is this change? It seems an unintentional change.

Reply:
https://github.com/cloudspannerecosystem/memefish/pull/310/commits/caa811ac16645ab6e8c1d79dffcb1937e0d8242c